inc/computer_softwareversion.class.php

```diff
@@ -656,42 +694,59 @@ static function showForComputer(Computer $comp, $withtemplate = 0) {
 
       $crit         = Session::getSavedOption(__CLASS__, 'criterion', -1);
 
-      $where        = '';
+      $where        = [];
+      $where_str    = '';
```

Comment:
I'd prefer to keep it here, because the _str only exists because of a lack in the iterator... Extra where clause should be the same in both queries, we'll miss in the future.